src/main/java/com/project/portfolio/springboot/onlinesellingandshopping/serverapp/service/UserService.java

```diff
@@ -0,0 +1,21 @@
+package com.project.portfolio.springboot.onlinesellingandshopping.serverapp.service;
+
+import com.project.portfolio.springboot.onlinesellingandshopping.serverapp.module.User;
+import com.project.portfolio.springboot.onlinesellingandshopping.serverapp.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+
```

Comment:
Needs Javadoc at the interface and methods level. See Effective Java Third Edition Item 56.